spec/features/visits/03_users_can_edit_a_visit_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+feature "users can edit visits within a vacation" do
+  scenario "signed in user can edit a visit they created" do
```

Comment:
Typically in RSPEC you define the variables you want to use before the scenario
here you could do `'let(:park) { FactoryGirl.create(park) }...`